website/plugins/proto-messages/index.js

```diff
@@ -102,47 +106,69 @@ module.exports = () => {
         // Discard the last line from the code block (pure newline).
         for (let i = 0, length = codeArray.length - 1; i < length; i++) {
           const line = codeArray[i];
+          let type = getTypeFromLine(line, namespace);
```

Comment:
This is message type or field type?

Reply:
It's the latter, yes. That's a good suggestion, thanks!